src/Bridge/Doctrine/EventListener/WriteListener.php

```diff
@@ -1,87 +0,0 @@
-<?php
```

Comment:
We cannot drop it right now (BC promise), this class must be deprecated instead.

Reply:
So should we also keep the definition of the tagged service and depreciate it?

Reply:
I'm not sure about that because it's a public service so it will always be executed.